assets/stylesheets/custom.scss

```diff
@@ -3,235 +3,255 @@ $red: #ae442e;
 $orange: orange;
 
 .good {
-	color: rgba(0, 255, 0, 0.4);
+  color: rgba(0, 255, 0, 0.4);
 }
 
 .bad {
-	color: rgba(255, 0, 0, 0.4);
+  color: rgba(255, 0, 0, 0.4);
 }
 
 h1 {
-	font-size: 22px;
+  font-size: 22px;
 }
 
 h1.header {
-	color: rgba(255,255,255,0.5);
-	font-weight: bold;
-	font-size: 30px;
+  color: rgba(255, 255, 255, 0.5);
+  font-size: 30px;
+  font-weight: bold;
 }
 
 h2 .value-add {
-	font-size: 0.4em;
-	vertical-align: middle;
+  font-size: 0.4em;
+  vertical-align: middle;
 }
 
 .trend-up {
-	color: $green;
+  color: $green;
 }
 
 .trend-down {
-	color: $red;
+  color: $red;
 }
 
 .more-info {
-	font-size: 12px;
-	opacity: 0.5;
+  font-size: 12px;
+  opacity: 0.5;
 }
 
 .highcontrast {
 
-	background-color: #222;
+  background-color: #222;
 
-	.widget {
-		color: white;
-		background-color: #333 !important;
-		padding: 0;
+  .widget {
+    color: white;
+    background-color: #333 !important;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used